lbry/dht/protocol/iterative_find.py

```diff
@@ -57,37 +56,19 @@ def get_close_triples(self) -> typing.List[typing.Tuple[bytes, str, int]]:
         return [(node_id, address.decode(), port) for node_id, address, port in self.close_triples]
 
 
-def get_shortlist(routing_table: 'TreeRoutingTable', key: bytes,
-                  shortlist: typing.Optional[typing.List['KademliaPeer']]) -> typing.List['KademliaPeer']:
-    """
-    If not provided, initialize the shortlist of peers to probe to the (up to) k closest peers in the routing table
-
-    :param routing_table: a TreeRoutingTable
-    :param key: a 48 byte hash
-    :param shortlist: optional manually provided shortlist, this is done during bootstrapping when there are no
-                      peers in the routing table. During bootstrap the shortlist is set to be the seed nodes.
-    """
-    if len(key) != constants.HASH_LENGTH:
-        raise ValueError("invalid key length: %i" % len(key))
```

Comment:
is this dropped intentionally?

Reply:
yes, it is checked on the constructor: https://github.com/lbryio/lbry-sdk/pull/3631/files#diff-6cdbfed907d98f6fa01d47ea54cdc27545a13fdcfa817760715c89e9c15d6376R64
(the shortlist comes from the caller now)